admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -359,15 +359,33 @@ var featuresCollection = (function() {
       /** On feature selector event change, refresh possible values list */
       $(document).on('change', '.feature-collection select.feature-selector', function() {
         var selector = $(this).parent().parent().parent().find('.feature-value-selector');
```

Comment:
`parent().parent().parent()` can be replaced with `parents('.feature-value-selector')` (returning an array of elements)
This `selector` variable can be confused with a jQuery selector, the variable could be renamed to `featureSelector`.